### PR TITLE
media-sound/helvum: new package, add 0.4.1

### PR DESCRIPTION
**Pull request by AdelKS**
Hello @thesamesam 

As requested, here's the helvum package from ::guru, feel free to request any change :D 

Adel

### CODE REVIEW

File: media-sound/helvum/helvum-0.4.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+# License set may be more restrictive as OR is not respected
+# use cargo-license for a more accurate license picture
+LICENSE="Apache-2.0 Apache-2.0-with-LLVM-exceptions BSD GPL-3 MIT Unicode-DFS-2016 Unlicense"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="mirror"
+
+BDEPEND="
+	sys-devel/clang
```
Comment (thesamesam) [inline media-sound/helvum/helvum-0.4.1.ebuild:130]: We need to use `llvm.eclass` for this - take a look at sequoia-* ebuilds for an example.

File: media-sound/helvum/helvum-0.4.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+# does not provide this value so instead repository is used
+HOMEPAGE="https://gitlab.freedesktop.org/pipewire/helvum"
+SRC_URI="
+	https://gitlab.freedesktop.org/pipewire/helvum/-/archive/${PV}/${P}.tar.bz2
+	${CARGO_CRATE_URIS}
+"
+
+# License set may be more restrictive as OR is not respected
+# use cargo-license for a more accurate license picture
+LICENSE="Apache-2.0 Apache-2.0-with-LLVM-exceptions BSD GPL-3 MIT Unicode-DFS-2016 Unlicense"
```
Comment (thesamesam) [inline media-sound/helvum/helvum-0.4.1.ebuild:124]: Please re-generate with pycargoebuild.

File: media-sound/helvum/helvum-0.4.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+	pipewire-0.7.0
+	pipewire-sys-0.7.0
+	pkg-config-0.3.27
+	proc-macro-crate-1.3.1
+	proc-macro-error-1.0.4
+	proc-macro-error-attr-1.0.4
+	proc-macro2-1.0.66
+	quote-1.0.33
+	regex-1.9.3
+	regex-automata-0.3.6
```
Comment (thesamesam) [inline media-sound/helvum/helvum-0.4.1.ebuild:81]: `pkgcheck scan` should warn on this as `-` as a separator is deprecated; please use pycargoebuild.

**Review by thesamesam — CHANGES_REQUESTED** [review]

File: media-sound/helvum/helvum-0.4.1.ebuild (hunk 3 above)
Comment (AdelKS) [reply media-sound/helvum/helvum-0.4.1.ebuild:81]: Yeah it did, I just did not want to bother at the time since I did not know about pycargoebuild :D

**Pull request closed without merging**